packages/gatsby-image/src/__tests__/index.js

```diff
@@ -55,12 +56,13 @@ describe(`<Img />`, () => {
     expect(component).toMatchSnapshot()
   })
 
-  it(`should have correct src, title and alt attributes`, () => {
+  it(`should have correct src, title, alt, and crossOrigin attributes`, () => {
     const imageTag = setup().querySelector(`picture img`)
     expect(imageTag.getAttribute(`src`)).toEqual(`test_image.jpg`)
     expect(imageTag.getAttribute(`srcSet`)).toEqual(`some srcSet`)
     expect(imageTag.getAttribute(`title`)).toEqual(`Title for the image`)
     expect(imageTag.getAttribute(`alt`)).toEqual(`Alt text for the image`)
+    expect(imageTag.getAttribute(`crossOrigin`)).toEqual(`anonymous`)
```

Comment:
i'll have to validate this still before merging, pretty sure it needs to be `crossorigin` instead of `crossOrigin`

Reply:
nvm getAttribute is case insensitive